scrapy/utils/conf.py

```diff
@@ -115,6 +115,10 @@ def feed_complete_default_values_from_settings(feed, settings):
     out = feed.copy()
     out.setdefault("encoding", settings["FEED_EXPORT_ENCODING"])
     out.setdefault("fields", settings.getlist("FEED_EXPORT_FIELDS") or None)
+    out.setdefault(
+        "batch_item_count",
+        out.get('batch_item_count', settings.getint('FEED_STORAGE_BATCH_ITEM_COUNT'))
+    )
```

Comment:
```suggestion
    out.setdefault("batch_item_count", settings.getint('FEED_STORAGE_BATCH_ITEM_COUNT')))
```

No need to check for the presence of a current value, `setdefault` will act only if it's missing.